pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -228,11 +231,11 @@ func (service *SecurityPolicyService) createOrUpdateGroups(nsxGroups []model.Gro
 		if err != nil {
```

Comment:
we don't see var err definition,  I think you neglect this var define, should put in the beginning of function.

Reply:
`func (service *SecurityPolicyService) createOrUpdateGroups(nsxGroups []model.Group) error {
	for _, group := range nsxGroups {
		group.MarkedForDelete = nil
		err := service.NSXClient.GroupClient.Patch(getDomain(service), *group.Id, group)
		if err != nil {
			return err
		}
	}
	
	err := service.groupStore.Operate(&nsxGroups)
	if err != nil {
		return err
	}
	log.Info("successfully create or update group", "groups", nsxGroups)
	return nil
}`